scripts/post-install.js

```diff
@@ -92,3 +103,110 @@ if (target.archive === 'tar.gz') {
 }
 
 console.log(' ✓ %s', outFile)
+
+//= == Zinnia runtime ===//
```

Comment:
Instead of adding structure comments, what do you think about moving the download logic into `lib/saturn.js` and `lib/zinnia.js`?

Reply:
Yeah, we can definitely move this code. I wanted to avoid code churn, but I guess it doesn't matter that much.

Another thing to consider: if we move this code into new files, then we need to update our Dockerfile to copy those files over.

Since this is in `scripts/post-install.js`, I am thinking about introducing `scripts/lib/download-zinnia.js` and `scripts/lib/download-saturn.js`. WDYT?

Reply:
Sounds all good to me! Docker should just copy over `script/*`

Reply:
Can probably be refactored using https://github.com/filecoin-station/core/pull/104/files#diff-17a4b8942ea9f50b9ecadea71f1fc85944e98028b12e0220d6d45c38513329c3

Reply:
If you think this lib will be useful, I can refactor it into a separate PR